client/asset/btc/btc.go

```diff
@@ -3299,7 +3303,7 @@ func (btc *baseWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin, ui
 	}
 
 	// Refund txs prepared and signed. Can now broadcast the swap(s).
-	err = btc.broadcastTx(msgTx)
+	_, err = btc.broadcastTx(msgTx)
```

Comment:
I'm totally fine with the `btc.broadcastTx` refactoring (I've thought about doing just this before), but this is the kind of thing I'd like to see in a separate commit in the future.  There are three lines in btc.go that pertain to what this PR aims to resolve.

Reply:
Workflow suggestion: experiment with `git add -p`, or similar "State Selected Ranges" functionality of some IDEs.  It helps add/stage only certain changes in a file.

Reply:
Yeah, totally agree. This one slipped through.